yb-voyager/cmd/importSchema.go

```diff
@@ -192,7 +209,46 @@ func importSchema() {
 	importSchemaCompleteEvent := createImportSchemaCompletedEvent()
 	controlPlane.ImportSchemaCompleted(&importSchemaCompleteEvent)
 
-	callhome.PackAndSendPayload(exportDir)
+	return nil
+}
+
+func packAndSendImportSchemaPayload(status string, errMsg string) {
```

Comment:
minor nit: some newlines in the function at meaningful points will increase the readability of the code.